tools/vscode/stl.code-workspace

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+{
+    "folders": [
+        {
+            "path": "../../stl"
+        }
+    ],
+    "settings": {
+        "editor.formatOnSave": true,

```

Comment:
YESSSSSSSSSSSSSSSSSSSSSS would prefer this be in the local "folder" settings instead of the workspace, so I don't forget it. This is the main source of "bad" formatting from me, I turn format on save off when working with a project that doesn't use clang-format and forget to turn it back on.